docs/api/regressor/peplot.rst

```diff
@@ -5,6 +5,13 @@ Prediction Error Plot
 
 A prediction error plot shows the actual targets from the dataset against the predicted values generated by our model. This allows us to see how much variance is in the model. Data scientists can diagnose regression models using this plot by comparing against the 45 degree line, where the prediction exactly matches the model.
 
+=================   ==============================
+Visualizer           `PredictionError <https://www.scikit-yb.org/en/latest/api/regressor/peplot.html>`_
+Quick Method         `prediction_error() <https://www.scikit-yb.org/en/latest/api/regressor/peplot.html#yellowbrick.regressor.peplot.prediction_errorr>`_
```

Comment:
It might be a tad easier if we use the sphinx autodoc member link syntax - this will ensure the links work for local builds, develop, latest, and our translations:

```
 Visualizer           :class:`~yellowbrick.regressor.residuals. PredictionError `
 Quick Method         :func:`~yellowbrick.regressor.residuals. prediction_error `
```

Note that the `~` in front means that it doesn't display the whole path, just the name of the function or class.